go-controller/pkg/util/net_linux.go

```diff
@@ -223,6 +223,25 @@ func LinkAddrFlush(link netlink.Link) error {
 	return nil
 }
 
+// LinkAddrDeleteMasqueradeIP() helps to remove stale V4HostMasqueradeIP or
```

Comment:
```suggestion
// LinkAddrDeleteMasqueradeIP removes stale V4HostMasqueradeIP or...
```
Also, in the context of this function `V4HostMasqueradeIP/V6HostMasqueradeIP` do not mean much. The function removes addresses that have the `<link-name>-masq` label set.

Reply:
yes, correct. I will modify the comment.